packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -67,14 +67,26 @@ export class BoosterTokenVerifier {
     const rolesClaim = this.config.tokenVerifier?.rolesClaim || 'custom:role'
     const role = decodedToken[rolesClaim]
     const roleValue = Array.isArray(role) ? role[0] : role
+    const customParams = this.getCustomParameters(decodedToken)
 
     return {
       id,
       username,
       role: roleValue?.trim() ?? '',
+      ...customParams,
     }
   }
 
+  private getCustomParameters(decodedToken: Record<string, unknown>): Record<string, unknown> {
+    return Object.keys(decodedToken)
+      .filter((key: string) => key.includes('custom:') && key !== 'custom:role')
```

Comment:
@juanjoman Take into account that the claim where the token is can be different. You should use the above variable (line 67) `rolesClaim`.

Anyway take a look at previous comment

Reply:
ooooh you're right, thanks!